deployment/ccip/changeset/test_helpers.go

```diff
@@ -2,6 +2,7 @@ package changeset
 
 import (
 	"context"
+	"errors"
```

Comment:
Looks like `pkg/errors` also has a `New` function, so no need to import both of these.